fe/endpoint_correction.py

```diff
@@ -18,12 +18,11 @@ def exp_u(rotation, k, beta):
 def sample_multiple_rotations(k, beta, size):
     num_batches = 500
     batch_size = 10000
-
+    state = np.random.RandomState(2021)
```

Comment:
Nit: Could possibly lead to subtle usage errors in the future if we later use `sample_multiple_rotations` in a new context that might assume independent rotations to be sampled on each call.

Might suggest either:
* adding a note in the docstring that it's a deterministic function of `(k, beta, size)`, or perhaps
* modifying the signature to include a `seed=None` or `random_state=None` optional argument, setting the seed deterministically from `estimate_delta_us`